docs/configuration/good-practices.rst

```diff
@@ -221,3 +205,66 @@ Scheduled down time
 ===================
 
 * Change Backoff setting in application configuration
+
+
+
+
+About sharding
+==============
+
+`Sharding <https://en.wikipedia.org/wiki/Shard_%28database_architecture%29>`_ is
+horizontal scaling, where the data is partitioned in different *shards*.
+
+A client is then automatically assigned a particular shard, depending for
+example:
+
+* on the request authorization headers
+* on the bucket or collection id
+
+It is currently not possible to setup the sharding directly from the kinto
+settings, however it is already possible to set it up manually. [#]_
+
+.. [#] http://www.craigkerstiens.com/2012/11/30/sharding-your-database/
+
+
+At the HTTP level
+-----------------
+
+It is possible to handle the sharding at the HTTP level. For instance, using
+a third-party service that assigns a node to a particular user.
+
+This has the advantage to be very flexible: new instances can be added and
+this service is in charge of partitioning, downside being maintaining a new
+service for it.
+
+The `tokenserver <https://github.com/mozilla-services/tokenserver>`_ is a good
+example of how sharding is done in Firefox Sync.
+
+The first time they connect, clients are asking the token server for a node, and
+then they talk directly with the node itself, without going trough the token
+server anymore, until a problem arises.
+
+At the load balancer level
+--------------------------
+
+The load balancer is the piece of software that takes all the requests upfront
+and route them to a different node, to make sure the load is equivalent on each
```

Comment:
The load balancer is the piece of software that takes

The load balancer is the piece of software that routes 

no ?